specs/toolsets/windows/setup-toolset.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
This should be merged in clang/setup-toolset.sh I believe, see the `pierre-linux-clang` branch (PR #10).